app/experimenter/experiments/migrations/0050_create_group_add_permissions.py

```diff
@@ -0,0 +1,45 @@
+from django.db import migrations
+
+
+def add_groups(apps, schema_editor):
+
+    Group = apps.get_model("auth", "Group")
+    Permission = apps.get_model("auth", "Permission")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    Experiment = apps.get_model("experiments", "Experiment")
+
+    relman_signoff_group = Group.objects.create(name="Relman Signoff")
+    qa_signoff_group = Group.objects.create(name="QA Signoff")
+
+    content_type = ContentType.objects.get_for_model(Experiment)
+    relman_permission = Permission.objects.get(
+        content_type=content_type, codename="can_check_relman_signoff"
+    )
+    qa_permission = Permission.objects.get(
+        content_type=content_type, codename="can_check_QA_signoff"
+    )
+
+    relman_signoff_group.permissions.add(relman_permission)
+    qa_signoff_group.permissions.add(qa_permission)
+
+
+def remove_groups(apps, schema_editor):
+    Group = apps.get_model("auth", "Group")
+
+    relman_signoff_group = Group.objects.get(name="Relman Signoff")
+    relman_signoff_group.delete()
+
+    qa_signoff_group = Group.objects.get(name="QA Signoff")
+    qa_signoff_group.delete()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("experiments", "0049_add_permissions"),
+        ("contenttypes", "0002_remove_content_type_name"),
+        ("experiments", "0048_experiment_risk_external_team_impact"),
+        ("auth", "0009_alter_user_last_name_max_length"),
+    ]
+
+    operations = [migrations.RunPython(add_groups, remove_groups)]
```

Comment:
The filename is "create_group_add_permissions", but it doesn't really explain what's going on here. Maybe something more descriptive like "create_relman_and_qa_groups"?